.env

```diff
@@ -1,3 +1,5 @@
 PORT=3001
 MONGODB_URI='mongodb://heroku_th7q4wxs:d5osftmugrhb70s389h6m8bcr4@ds133476.mlab.com:33476/heroku_th7q4wxs'
-REACT_APP_APIURL='https://bears-20-dev.herokuapp.com/api'
\ No newline at end of file
+REACT_APP_APIURL='https://bears-20-dev.herokuapp.com/api'
+GITHUB_CLIENT_ID='aaa8f183a57185dedf84'
+GITHUB_CLIENT_SECRET='3c47ead1b412eb21df2c22f2bc747620be349bd3'
```

Comment:
Is it safe to send .env file on github?? Maybe we should add .env to .gitignore and send to github repo file 
.env-example   
with:
... 
+GITHUB_CLIENT_ID='your client id goes here...'
+GITHUB_CLIENT_SECRET='your client secret goes here...'
I just don't know how important this info is ?

Reply:
Yep definitely we should gitignore the .env file, especially now that we have some sensitive info in there. My thinking was to have it in there just until everyone has their dev environment setup locally.